src/layout/Summary/SummaryComponent.tsx

```diff
@@ -121,7 +113,7 @@ export function SummaryComponent({ summaryNode, overrides }: ISummaryComponent)
           <SummaryContent
             onChangeClick={onChangeClick}
             changeText={changeText}
-            label={label}
+            label={langAsString(titleKey)}
```

Comment:
Just noticed that this should be `lang` instead of `langAsString`. This breaks the formatting. The old method used getTextFromAppOrDefault with `stringOutput=false`.

![image](https://github.com/Altinn/app-frontend-react/assets/47412359/f0a3f03e-06b5-45bc-8835-a29927d72cd4)